BasicAlgorithmsTests/BubbleSortTest.cs

```diff
@@ -0,0 +1,33 @@
+﻿using System;
+using BasicAlgorithms;
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+
+namespace BasicAlgorithmsTests
+{
+    [TestClass]
+    public class BubbleSortTest
+    {
+        //Given When Then
+        [TestMethod]
+        public void ArrayWithDuplicatesElem_Sort_ArrayAscendingSort()
+        {
+
+            int[] unsortedList = new int[] {2, 2, 1};
+            int[] sortedList = new int[] { 1, 2, 2 };
```

Comment:
A good naming can be something like actual = the result of what I am doing and expected = how I suppose it will look like